public/directives/wz-logtest/components/logtest.tsx

```diff
@@ -124,18 +147,41 @@ export const Logtest = compose(
           onKeyPress={handleKeyPress}
         />
         <EuiSpacer size="m" />
+        <EuiFlexGroup justifyContent="spaceBetween">
         <EuiFlexItem grow={false}>
           <EuiButton
             style={{ maxWidth: '100px' }}
             isLoading={testing}
-            isDisabled={testing || value.length === 0}
+            isDisabled={testing || events.length === 0}
             iconType="play"
             fill
             onClick={runAllTests}
           >
             Test
           </EuiButton>
         </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <WzButtonPermissionsModalConfirm
+            style={{ maxWidth: '150px' }}
+            tooltip={{position: 'top', content: 'Clear current session'}}
+            fill
+            isDisabled={sessionToken === '' ? true : false}
+            aria-label="Clear current session"
+            iconType="broom"
+            onConfirm={async () => {
+              deleteToken();
+            }}
+            color="danger"
+            modalTitle={`Do you want to clear current session?`}
+            modalProps={{
+              buttonColor: 'danger',
+              children: 'Clearing the session means the logs execution history is removed. This affects to rules that fire an alert when similar logs are executed in a specific range of time.'
+            }}
+          > 
+          Clear sesion 
```

Comment:
Typeerror it should be `session` instead of sesion

Reply:
I have already modified it, thanks you!